.happy/terraform/envs/stage/main.tf

```diff
@@ -13,7 +13,7 @@ module stack {
   frontend_url                 = "https://cellxgene.staging.single-cell.czi.technology"
   backend_url                  = "https://api.cellxgene.staging.single-cell.czi.technology"
   stack_prefix                 = ""
-  batch_container_memory_limit = 300000
+  batch_container_memory_limit = 60000
```

Comment:
Should be enough for the staging cube

Reply:
If it's not too onerous, I would keep staging and prod settings in sync. We do intend to be able to mirror prod data to other envs at some point. Not sure if we're intending to do that for all or a subset of data though.

Reply:
Not too onerous, after all this is just a memory requirement and the staging batch environment has enough to satisfy 300G. If we decide to mirror the prod data, then it definitely makes sense to keep it at 300G.